src/workloads/query/TenMDocCollection_IntId.yml

```diff
@@ -111,9 +111,9 @@ Actors:
   - ActorFromTemplate:
       TemplateName: CrudTemplate
       TemplateParameters:
-        Name: IDFieldPointProject128Threads
+        Name: IDHackEqPointProject128Threads
```

Comment:
Does this rename cause the history of the test to be lost/messed up?

Reply:
If you rename it the history will be lost because this will be a "new" metric. But taking into account that the query optimisation is now completely different (IDHACK) and therefore the performance is not comparable, I do not think the history really matters in this case.